### PR TITLE
feat: update deps

### PR DESCRIPTION
**Pull request by simboonlong**
-  `@govtechsg/open-attestation` 5.2.0 upwards, causing testcafe 1 test to fail (keeping to 5.1.3 for now)
- guessing that is due to some tsconfig configuration that i'm missing out (i'm stuck for way too long for this), which is causing the typescript compile error
- the error happens right at `import { v2 } from "govtechsg/open-attestation";`
- i could rewrite the test to pass it, but i think i shouldn't for this case

`@govtechsg/open-attestation` 5.3.2 gives this when `npm run integration`:

Screenshot 2021-06-07 at 3 34 35 PM

### CODE REVIEW

File: src/templates/customTemplate/customTemplate.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
+/// <reference types="@emotion/react/types/css-prop" />
```
Comment (Nebulis) [inline src/templates/customTemplate/customTemplate.tsx:1]: should be able to configure to not need that :thinking:

File: jest.setup.js [hunk 2]
```diff
@@ -1 +1,3 @@
-global.fetch = require("whatwg-fetch").fetch;
+global.fetch = require("jest-fetch-mock");
```
Comment (Nebulis) [inline jest.setup.js:1]: hum why ?

Comment (simboonlong) [reply jest.setup.js:1]: im getting some weird fetch error in circle ci `Jest did not exit one second after the test run has completed`, was trying this to see if it works, but doesnt seem so too :(

**Comment by simboonlong** [comment]
stale, closing

**Pull request closed without merging**